src/Toolkit/Toolkit.iOS/UI/Controls/TimeSlider/TimeSlider.iOS.cs

```diff
@@ -221,9 +228,14 @@ private void InitializeImpl()
             Tickmarks = new Primitives.Tickbar()
             {
                 TickFill = TimeStepIntervalTickFill,
-                TickLabelColor = TimeStepIntervalLabelColor,
                 ShowTickLabels = LabelMode == TimeSliderLabelMode.TimeStepInterval,
             };
+
+            if (TimeStepIntervalLabelColor != null)
+            {
+                Tickmarks.TickLabelColor = TimeStepIntervalLabelColor;
```

Comment:
If TimeStepIntervalLabelColor is a nullable property, should TickLabelColor also be nullable?